services/web/server/src/simcore_service_webserver/director_v2_core.py

```diff
@@ -439,3 +441,27 @@ async def retrieve(
 
     assert isinstance(retry_result, dict)  # nosec
     return retry_result
+
+
+@log_decorator(logger=log)
+async def restart(app: web.Application, node_uuid: str) -> None:
+    # when triggering retrieve endpoint
+    # this will allow to sava bigger datasets from the services
+    timeout = ServicesCommonSettings().restart_containers_timeout
```

Comment:
why this setting is not created under the app settings? The idea is that all the settings are captured at the app init and then frozen.This way we can have a better control of the variables involved in the config of a service. 
If we capture environs everywhere we go back to the previous setup where we do not really know which variables we have.

Reply:
You can see that this is a very old one. I think we can put it as a Friday refactor case: removing the settings out of models-library.